Testscripts/Windows/VERIFY-INFINIBAND-MultiVM.ps1

```diff
@@ -141,6 +141,13 @@ function Main {
 			if ($TestParam -imatch "mpi_type") {
```

Comment:
Following on the comment below about improving the framework this would read much better as
$MpiType = GetStringTestParam "mpi_type"

Reply:
@friver-ms  I added you for the reference, but you put some valuable comment. Can you open new PR on your name? We can work on the improvement. Sorry for the confusion, but PR merged.